tests/apps/study/test_study_api.py

```diff
@@ -56,3 +57,77 @@ def test_study_detail_api(db_keys):
         "url": "/study/1/",
         "rob_response_values": [17, 16, 15, 12, 14, 10],
     }
+
+
+@pytest.mark.django_db
+class TestStudyCreateApi:
+    def test_permissions(self, db_keys):
+        url = "/study/api/study/"
+        data = {
+            "reference_id": db_keys.reference_unlinked,
+            "short_citation": "Short citation.",
+            "full_citation": "Full citation.",
+        }
+
+        # reviewers shouldn't be able to create
+        client = APIClient()
+        assert client.login(username="rev@rev.com", password="pw") is True
+        response = client.post(url, data)
+        assert response.status_code == 403
+
+        # public shouldn't be able to create
+        client = APIClient()
+        response = client.post(url, data)
+        assert response.status_code == 403
+
+    def test_bad_requests(self, db_keys):
+        # payload needs to include the required short_citation and full_citation
+        url = "/study/api/study/"
```

Comment:
instead of hard coding, use `reverse("study:api:study-list")` here and elsewhere